pages/05.android/06.solving-problems/09.background-work/docs.en.md

```diff
@@ -100,4 +102,22 @@ For Samsung devices, there is no great need for setting up background work, but
 
 <img src="https://github.com/TheHasagi/AdguardKnowledgeBase/blob/master/pages/05.android/06.solving-problems/09.background-work/Images/Oneplus.png?raw=true" width="300">
 
+<a id="Google"></a>
+
+## Android stock devices Pixel/Nexus/Essential
+
+Android stock OS is not conflict with background work of applications, but if you are facing the issues, you will need to switch on "Always on VPN" mode that help you.
```

Comment:
Android stock OS normally does not conflict with applications working in background, but if you are facing any issues you will need to switch on the "Always on VPN" mode.